ci/ci/build.py

```diff
@@ -315,16 +318,26 @@ def build(self, batch, code, scope):
 
 mv /python3.7-slim-stretch/Dockerfile.out {shq(dockerfile_in_context)}
 
-set +e
-/busybox/sh /convert-google-application-credentials-to-kaniko-auth-config
-set -e
+set +x
+/bin/sh /home/user/convert-google-application-credentials-to-docker-auth-config
+set -x
 
-exec /kaniko/executor --dockerfile={shq(dockerfile_in_context)} --context=dir://{shq(context)} --destination={shq(self.image)} --cache=true --cache-repo={shq(cache_repo)} --snapshotMode=redo --use-new-run'''
+export BUILDKITD_FLAGS=--oci-worker-no-process-sandbox
+buildctl-daemonless.sh \
```

Comment:
it's unclear to me where this script comes from.

Reply:
Ah, my bad. I missed the very first commit in this series when I ripped it out of my wider batch changes.

I replaced `KANIKO_IMAGE` and `ci/kaniko/` with `BUILDKIT_IMAGE` and `ci/buildkit`. `buildctl-daemonless.sh` is part of the buildkit image.